src/pages/Pokemons/components/PokemonsView/index.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+import { Button } from "@mui/material";
+import { Link } from "react-router-dom";
+import { v4 as uuid } from "uuid";
+
+import { ROUTE_NAMES } from "Routes/routeNames";
+
+import styles from "./style.module.scss";
+
+const PokemonsView = ({ pokemons, handleGetPokemon }) => {
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.pokemonGretings}>
+        <h1>Pokemons</h1>
+        <p>Click button get pokemons.</p>
+        <Button variant="contained" onClick={handleGetPokemon}>
+          GO
+        </Button>
+      </div>
+      <div className={styles.pokemonList}>
+        {pokemons.map(({ name }) => (
+          <Link to={`${ROUTE_NAMES.POKEMONS}/${name}`} key={uuid()}>
```

Comment:
don't use uuid() like that. I told about it on examples of this issue. in your case you can use name as a key